message tab_characterUnitsTable 
{
	optional string tname = 1;
	repeated tab_characterUnits tlist = 2;
}

message tab_characterUnits
{
	optional int32 id = 1;
	optional string Comment = 2;
	optional int32 UnitCount = 3;
}